docs/blog/2022-05-02-webops.md

```diff
@@ -0,0 +1,24 @@
+---
+description: CMS previews now require authentication by default
+tags: [webops]
+---
+
+# CMS previews now require authentication by default
+
+Following the next successful push to your store's repository default branch, the CMS preview available at `https://gatsby-preview-vtex-sites--<your-account-here>-store.vtex.io` will require users to authenticate to VTEX.
+
+In order to access the preview, users will need to have access to the `myvtex.com` domain of that specific account. This can be done either through the build environment variable `GATSBY_STORE_ID` in the `vtex.env` file, or the `api.storeId` entry exported by the `store.config.js` configuration file.
+
+## What has changed?
+
+Previously, CMS previews were published to public URLs by default. Now, they are private by default, requiring authentication to VTEX.
+
+## Why did we make these changes?
+
+It is common for sales promotions and product launches to appear in CMS previews before they are made available to the public. This change aims to prevent that while making the platform more secure.
```

Comment:
```suggestion
CMS previews may contain sales promotions and product launches that are not yet available to the general public. This change aims to prevent unauthorized access to such content while also making the platform more secure.
```

Reply:
unauthorized, undesired...